context/index.js

```diff
@@ -23,9 +24,30 @@ export const AppContextProvider = (props) => {
     });
   };
 
+  //OpenDB Token so we do not get the same question twice, sessions are deleted after 6 hours
+  useEffect(() => {
+    const getToken = async () => {
+      Axios.get('https://opentdb.com/api_token.php?command=request').then(
+        (res) => {
+          if (res.data.response_code === 0) {
+            setToken(res.data.token);
+          } else {
+            Axios.get(
+              `https://opentdb.com/api_token.php?command=reset&token=${token}`,
+            ).then((response) => {
+              setToken(response.data.token);
+            });
+          }
+        },
+      );
+    };
+    getToken();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I added this in because adding in the reco dep [token] causes an infinite loop.

Reply:
Try adding this rule to the eslint config file, I've seen it in previous projects and was unsure what I did - may be for this situation? 

``` 
module.exports = {
  root: true,
  extends: '@react-native-community',
  rules: {
    'react-hooks/exhaustive-deps': 0,
  },
};
```

Reply:
done, that worked